sandbox/cgroup/cgroup.go

```diff
@@ -263,18 +263,21 @@ func ProcessPathInTrackingCgroup(pid int) (string, error) {
 	// It will forever stay present in the kernel but will not be present in
 	// the file-system. As such, allow v2 to register only if it is really
 	// mounted on the system.
```

Comment:
I suppose the comment needs to be updated to be more explicit about the new approach?

Reply:
Right, also the comment (I'm not sure if it was this or another one) was saying that this is a behaviour specific to cgroup v2, whereas the Linux kernel documentation (and this experience) says that it also affects v1. I'll update the comment.